maskrcnn_benchmark/structures/segmentation_mask.py

```diff
@@ -361,8 +361,9 @@ def __init__(self, polygons, size):
         self.polygons = []
         for p in polygons:
             p = PolygonInstance(p, size)
-            if len(p) > 0:
-                self.polygons.append(p)
+            #  if len(p) > 0:
```

Comment:
Why do we have this change?

Reply:
Hi, 

I believe this modification anymore is no longer needed any more. I modified this because our json files have empty `segmentation` field in the `annotations` part with will break the logic here.

Reply:
I have re-committed this pr, and new link is [here](https://github.com/facebookresearch/maskrcnn-benchmark/pull/845). If you please, we could continue communicating in that page :)